src/stdbuf/libstdbuf/libstdbuf.rs

```diff
@@ -55,6 +55,8 @@ fn set_buffer(stream: *mut FILE, value: &str) {
     }
 }
 
+/// # Safety
+/// ToDO ... (safety note)
```

Comment:
This should be perfectly safe AFAIK, as the only `unsafe` is from the macro (which just returns `stdin`/`stdout`/`stderr`) and from `setvbuf` in `set_buffer`.  The biggest concern I guess is if functions with the `constructor` attribute can allocate (I think they can?), because if they cannot `env::var` and stuff will be an issue.

Reply:
I'm not well-versed enough in `unsafe` code to create the comment. I'm happy to include whatever text you'd like ...